code/__DEFINES/status_effects.dm

```diff
@@ -200,3 +218,8 @@
 #define adjust_static_vision_up_to(duration, up_to) adjust_timed_status_effect(duration, /datum/status_effect/static_vision, up_to)
 #define set_static_vision(duration) set_timed_status_effect(duration, /datum/status_effect/static_vision)
 #define set_static_vision_if_lower(duration) set_timed_status_effect(duration, /datum/status_effect/static_vision, TRUE)
+
+#define adjust_drunkeness(strength) adjust_status_effect_strength(/datum/status_effect/inebriated, strength)
+#define adjust_drunkeness_up_to(strength, up_to) adjust_status_effect_strength(/datum/status_effect/inebriated, strength, 0,  up_to)
+#define set_drunkeness(strength) set_status_effect_strength(/datum/status_effect/inebriated, strength)
+#define get_drunkeness get_status_effect_strength(/datum/status_effect/inebriated)
```

Comment:
As per our contribution guide, this should be invoked like a proc -
```suggestion
#define get_drunkeness(...) get_status_effect_strength(/datum/status_effect/inebriated)
```
- so you type `mob.get_drunkness()` and not `1`mob.get_drunkess`

Reply:
Yeah I wondered how you did thant. Thanks!